src/Yoti.Auth/Profile/YotiProfile.cs

```diff
@@ -40,7 +40,9 @@ public YotiAttribute<Image> Selfie
         }
 
         /// <summary>
-        /// FullName represents the user's full name. This will be null if not provided by Yoti.
+        /// FullName represents the user's full name. If family_name/given_names are present, the
+        /// value will be equal to the string 'given_names + " " family_name'. Will be nil if not
```

Comment:
```suggestion
        /// value will be equal to the string 'given_names + " " + family_name'. Will be nil if not
```

Should there be an extra `+` here?